std/stdio.d

```diff
@@ -1075,21 +1075,37 @@ Throws: `ErrnoException` if the file is not opened or if the call to `fwrite` fa
 
         version (Windows)
         {
-            flush(); // before changing translation mode
             immutable fd = ._fileno(_p.handle);
-            immutable mode = ._setmode(fd, _O_BINARY);
-            scope(exit) ._setmode(fd, mode);
+            immutable oldMode = ._setmode(fd, _O_BINARY);
+            
```

Comment:
trailing whitespace line 1080 here (according to autotester)

Reply:
Should be good, assuming my editor did its job since I'm incapable of discerning needless whitespace.